Cython/Compiler/ExprNodes.py

```diff
@@ -6601,11 +6601,13 @@ def attribute_is_likely_method(attr):
         code.globalstate.use_utility_code(
             UtilityCode.load_cached("PyObjectFastCall", "ObjectHandling.c"))
 
+        padding_null_arg = code.funcstate.allocate_temp(py_object_type, manage_ref=True)
+        code.putln("%s = NULL;" % padding_null_arg)
         code.putln("{")
         code.putln("PyObject *__pyx_callargs[%d] = {%s, %s};" % (
-            len(args)+1,
+            len(args)+2,
             self_arg,
-            ', '.join(arg.py_result() for arg in args)))
+            ', '.join(tuple(arg.py_result() for arg in args) + (padding_null_arg,))))
```

Comment:
I think you could just do

```
            ', '.join(tuple(arg.py_result() for arg in args) + ("NULL",))))
```

Not sure you need to allocate a temp. And it definitely doesn't need refcounting